nilearn/plotting/html_surface.py

```diff
@@ -0,0 +1,584 @@
+                        info["_" + attribute], "int32");
+                }
+            }
+
+            info["vertexcolor"] = surfaceMapInfo["vertexcolor_" + hemisphere];
+
+            let data = [info];
+
+            info['lighting'] = getLighting();
+            let layout = getLayout();
+            let config = getConfig();
+
+            Plotly.react(divId, data, layout, config);
+
+            addColorbar(divId, layout, config);
+
+        }
+    </script>
+    <script>
+        $(document).ready(
+            function() {
+                if(surfaceMapInfo["full_brain_mesh"]){
+                    $("#select-hemisphere").change(addPlot);
+                    $("#select-kind").change(addPlot);
+                }
+                else{
+                    $("#select-hemisphere").hide();
+                    $("#select-kind").hide();
+                }
+                addPlot();
+                $("#select-view").change(updateLayout);
+                $("#surface-plot").mouseup(function() {
+                    $("#select-view").val("custom");
+                });
+                $(window).resize(updateLayout);
+
+            });
+    </script>
+</head>
+
+<body>
+    <div id="surface-plot"></div>
+    <select id="select-hemisphere">
+<option value="left">Left hemisphere</option>
+<option value="right">Right hemisphere</option>
+</select>
+
+    <select id="select-kind">
+<option value="inflated">Inflated</option>
+<option value="pial">Pial</option>
+</select>
+    <select id="select-view">
+<option value="left">view: Left</option>
+<option value="right">view: Right</option>
+<option value="front">view: Front</option>
+<option value="back">view: Back</option>
+<option value="top">view: Top</option>
+<option value="bottom">view: Bottom</option>
+<option value="custom">view: -</option>
+</select>
+
+</body>
+
+</html>
+
+"""
+
+
+def add_js_lib(html, embed_js=True):
+    if not embed_js:
+        js_lib = """
+        <script
+        src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js">
+        </script>
+        <script src="https://cdn.plot.ly/plotly-gl3d-latest.min.js"></script>
+        """
+    else:
+        js_dir = os.path.join(os.path.dirname(__file__), 'data', 'js')
+        with open(os.path.join(js_dir, 'jquery.min.js')) as f:
+            jquery = f.read()
+        with open(os.path.join(js_dir, 'plotly-gl3d-latest.min.js')) as f:
+            plotly = f.read()
+        js_lib = '<script>{}</script>\n<script>{}</script>'.format(
+            jquery, plotly)
+    return html.replace('INSERT_JS_LIBRARIES_HERE', js_lib)
+
+
+class HTMLDocument(object):
+    """
+    Represents a web page.
+
+    use str(document) or document.html to get a web page,
+    document.iframe() to have it wrapped in an iframe.
+
+    """
+
+    def __init__(self, html, width=600, height=400):
+        self.html = html
+        self.width = width
+        self.height = height
+
+    def iframe(self, width=None, height=None):
```

Comment:
Maybe "get_iframe"?

Also, as this is a public method, it should have a docstring.